api/python/t4/packages.py

```diff
@@ -406,6 +406,21 @@ def walk(self):
                 for key, value in child.walk():
                     yield name + '/' + key, value
 
+    def _walk_dir_meta(self):
```

Comment:
wouldn't we want to implement this on top of `walk()`?

Reply:
I don't know about that, it's at about the same level of abstraction as `walk()` but it does something different. I'm not sure how it'd be done on top of `walk()`